utils/namespaces.js

```diff
@@ -0,0 +1,33 @@
+import { sortBy } from '@/utils/sort';
+
+// This method is used by the NameNsDescription component and by the create/edit
+// workload form to provide namespaces to choose from.
+const getAvailableNamespaces = (resource) => {
```

Comment:
Resource is actually the whole Vue instance which you are passing here. It's usually not convenient to pass such an huge object for your function.
It may still be cumbersome even to pass the getters with `resource.$store.getters`.
In an util you may not want to have anything of this logic and rather work with the entries, especially since we should not have these getters even in the components in a first place.

In a classic Vuex pattern, this whole logic should be converted in a single getter which can be used directly from the View, rather than an helper/util.
I am not so sure what are the guidelines for this very project, but using Vuex we should have be free to at least follow the original patterns.

If this is not possible though, or you prefer postpone this to a separated talk, please discard this comment and consider on reducing the passed argument whether you agree.

Reply:
> In a classic Vuex pattern, this whole logic should be converted in a single getter which can be used directly from the View, rather than an helper/util.

I'm not sure what you mean - do you have an example?

Reply:
As in the documentation about chain getters: https://vuex.vuejs.org/guide/getters.html#property-style-access